.github/workflows/periodic-integration-tests.yml

```diff
@@ -79,8 +79,7 @@ jobs:
         working-directory: integration_tests/sdk
         env:
           SERVER_ADDRESS: localhost:8080
-          INTEGRATION: aqueduct_demo
-        run: pytest . -rP -n ${{ matrix.concurrency }}
+        run: pytest aqueduct_tests/ -rP -n ${{ matrix.concurrency }}
```

Comment:
why don't we run data integration tests here?

Reply:
The periodic integration tests will be improved as part of the automation milestone. I just didn't bother adding them here since the these tests currently contribute very little and are destined to be replaced soon.